### ISSUE TITLE
$(...).froalaEditor is not a function

### ISSUE THREAD

**Issue by reggin** [froala/KMSFroalaEditorBundle#9]
Hi
Just installed your bundle. I've add configuration, add froala to form (a form field type). And have error `$(...).froalaEditor is not a function`. 
`froala_editor.min.js` loaded also. What could be the problem?

**Comment by samarhxc** [comment]
Hi, 
do you manually loads javascript or let the bundle include it ?

**Comment by reggin** [comment]
Hi,
Sorry for delay. I let bundle to include it. Now I tried to include manually - and it works now, thanks!

**Comment by Jakub-Fajkus** [comment]
Hello,
I have the same issue. I let the bundle to include all required files. How can I at least include files manually?

**Comment by reggin** [comment]
@Jakub-Fajkus 
Hardcode :)
`<script src="{{ asset( 'bundles/kmsfroalaeditor/froala_editor_2.1.0/js/froala_editor.min.js' ) }}"></script>`

The same I've added plugins and styles.

**Comment by samarhxc** [comment]
I'll take a look and maybe ask you some questions about your configuration, libraries...

**Comment by Jakub-Fajkus** [comment]
I think I found the problem.
I am trying this on dev environment which is much slower than production. On production, it works without problem. I suppose the problem was because the jQuery's function ready() got triggered before the "froala_editor.min.js" file was fully loaded.

**Comment by pedrocasado** [comment]
I used to load all javascripts at the botton of the page and was having the same problem.. 

I just had to put jQuery js on the <head> tag and everything loaded correctly.

**Comment by helmidridi** [comment]
I have the same problem too

**Comment by dcognata** [comment]
Same problem

**Comment by samarhxc** [comment]
Yes, sometimes javascript files are not fully loaded, or some plugins are missing, need to investigate..

**Comment by gacel112** [comment]
I have the same problem.

**Comment by jk-** [comment]
Samething happened to me. I put includeJS to false in the config.yml and then included it myself. That solved the problem.

**Comment by gacel112** [comment]
Me too.

**Comment by krisElphenor** [comment]
Same problem for me. But even after including the js manually, i still got the same error.

**Comment by gacel112** [comment]
I solved, "Samething happened to me. I put includeJS to false in the
config.yml and then included it myself. That solved the problem."

2016-12-14 17:25 GMT+01:00 krisElphenor <notifications@github.com>:

**Comment by krisElphenor** [comment]
@gacel112 Yes, I tried that as well, no luck so far...still getting the same error

**Comment by krisElphenor** [comment]
I solved it by using the getScript function of jQuery.

`$.getScript("/path/to/js/froala_editor.min.js", function () {
    $( "textarea" ).froalaEditor({
        ...
    });
});`

Apparently, for me as well, the problem is the js not being fully loaded when the function froalaEditor is called. But i cant't explain why, even after loading the script manually both in the header and at the bottom of the page, I still get the same issue.

So, if someone else has the same problem, jquery getscript could be your 'plan C'

**Comment by somoza** [comment]
Same here... No solution jet?

**Comment by pushand** [comment]
Why don't you guys start with simple JS integration guide and then start with bundle guide

**Comment by seddighi78** [comment]
Same problem ...

**Comment by skamsie** [comment]
having the same problem and none of these suggestions work

**Comment by Levii01** [comment]
Also same

**Comment by dev-vishnupundhir** [comment]
same issue

**Comment by Mifrill** [comment]
any updates here?
